app/models/lead_provider.rb

```diff
@@ -1,27 +1,27 @@
 class LeadProvider < ApplicationRecord
   ALL_PROVIDERS = {
     "Ambition Institute" => "9e35e998-c63b-4136-89c4-e9e18ddde0ea",
-    "Best Practice Network (home of Outstanding Leaders Partnership)" => "57ba9e86-559f-4ff4-a6d2-4610c7259b67",
+    "Best Practice Network" => "57ba9e86-559f-4ff4-a6d2-4610c7259b67",
     "Church of England" => "79cb41ca-cb6d-405c-b52c-b6f7c752388d",
     "Education Development Trust" => "21e61f53-9b34-4384-a8f5-d8224dbf946d",
     "LLSE" => "230e67c0-071a-4a48-9673-9d043d456281",
     "National Institute of Teaching" => "3ec607f2-7a3a-421f-9f1a-9aca8a634aeb",
     "School-Led Network" => "bc5e4e37-1d64-4149-a06b-ad10d3c55fd0",
     "Teacher Development Trust" => "30fd937e-b93c-4f81-8fff-3c27544193f1",
     "Teach First" => "a02ae582-f939-462f-90bc-cebf20fa8473",
-    "UCL Institute of Education" => "ef687b3d-c1c0-4566-a295-16d6fa5d0fa7",
+    "University College London (UCL) Institute of Education" => "ef687b3d-c1c0-4566-a295-16d6fa5d0fa7",
```

Comment:
this change is interesting, I was hoping we could keep it consistent with ECF which is `UCL Institute of Education`, unless we plan to change it there as well?

Reply:
@cwrw and I discussed this and it was decided to accept the name inconsistency in NPQ/ECF for now. It can be updated after the NPQ separation MVP work is completed.